examples/std_web/todomvc/static/styles.css

```diff
@@ -0,0 +1,521 @@
+html,
```

Comment:
Do we need this still?

Reply:
We'd need a different method for pull in the css in the showcase `Model`